modules/common/util/auto_lock.h

```diff
@@ -0,0 +1,88 @@
+/******************************************************************************
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
```

Comment:
2019

Reply:
OK, thank you for your remind. I will fix this issue later.